js/plugins/ollama/src/embeddings.ts

```diff
@@ -76,12 +79,12 @@ export function defineOllamaEmbedder(
       },
     },
     async (input, config) => {
-      const serverAddress = config?.serverAddress || options.serverAddress;
+      const serverAddress = options.serverAddress || 'http://localhost:11434';
```

Comment:
We're no longer getting serverAddress from config, because (as far as i understand) it doesn't exist on that interface any more in v2 API.

Can we mention this in the PR description